CMakeLists.txt

```diff
@@ -47,21 +47,29 @@ if(NOT CMAKE_BUILD_TYPE)
 endif()
 
 # Version tokens
-include(GetGitRevisionDescription)
-get_git_head_revision(GIT_REFSPEC NVIM_VERSION_COMMIT)
-if(NOT NVIM_VERSION_COMMIT)
-  set(NVIM_VERSION_COMMIT "?")
-endif()
 set(NVIM_VERSION_MAJOR 0)
-set(NVIM_VERSION_MINOR 0)
+set(NVIM_VERSION_MINOR 1)
 set(NVIM_VERSION_PATCH 0)
 set(NVIM_VERSION_PRERELEASE "-alpha")
-# TODO(justinmk): UTC time would be nice here #1071
-git_timestamp(GIT_TIMESTAMP)
-# TODO(justinmk): do not set this for "release" builds #1071
```

Comment:
Dear Florian Walch,

The email sent to danielm@tinyhub.tk with subject
"Re: [neovim] [WIP] CMake: Prepare for releases. (#3318)"
will be delivered as soon as possible.

Your email address is automatically added to the
recipients 'whitelist'. This means for further
emails to danielm@tinyhub.tk, you will not receive a delivery
notification again.

TK MAILIAS
*************************************************
Looking for a way to Protect Your Identity and 
for a great no-more-spam solution for your email
address reply@reply.github.com? Sign up for TK Mailias.

It's free! Please visit http://www.mail.tk/